feature/user_profile/presentation/src/main/java/com/sanaa/presentation/screen/myRating/component/RatedMediaListSectionContent.kt

```diff
@@ -0,0 +1,63 @@
+package com.sanaa.presentation.screen.myRating.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.navigationBarsPadding
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.sanaa.designsystem.design_system.component.chips.ToggleableChip
+import com.sanaa.presentation.screen.myRating.MediaTypeUi
+import com.sanaa.presentation.screen.myRating.MyRatingScreenUiState
+import com.sanaa.presentation.screen.myRating.MyRatingTab
+
+@Composable
+fun RatedMediaListSectionContent(
+    state: MyRatingScreenUiState,
+    onTabSelected: (MyRatingTab) -> Unit,
+    onDeleteRatingClick: (mediaId: Int, mediaType: MediaTypeUi) -> Unit,
+    onCardClick: (mediaId: Int, mediaType: MediaTypeUi) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier.navigationBarsPadding(),
+        verticalArrangement = Arrangement.spacedBy(12.dp)
+    ) {
+
+        LazyRow(
+            contentPadding = PaddingValues(horizontal = 16.dp),
+            horizontalArrangement = Arrangement.spacedBy(8.dp),
+        ) {
+            items(items = MyRatingTab.entries.toTypedArray()) { tab ->
+                val tabText = when (tab) {
+                    MyRatingTab.ALL -> "All"
```

Comment:
get the string value from a resource 
```suggestion
                    MyRatingTab.ALL -> stringResource(R.string.all)
```

Reply:
also, you can add a variable for the tab label inside `MyRatingTab `enum class and use it instead the when statement